merge_sort/Cpp/merge_sort.cpp

```diff
@@ -0,0 +1,73 @@
+#include <iostream>
+#include <cstdio>
+#include <vector>
+using namespace std;
+std::vector<int> vec;
+
+void join(std::vector<int>& arr, int, int, int);
+void sort(std::vector<int>& arr, int, int);
+void show(std::vector<int>& arr);
+
+int main() {
+	int n = 10, a, i, arr[] = {2, 4, 3, 6, 5, 1, 9, 0, 8, 7};
+	// scanf("%d", &n); //For custom input
+	// printf("Enter %ld number (space separated): ", n); //For custom inup
+	for (i = 0; i < n; i++) {
+		//scanf("%d", &a);
+		vec.push_back(arr[i]); //vec.push_back(a); // For custom input
+	}
+	a = 0;
+	sort(vec, a, n-1);
+	show(vec);
+	return 0;
+}
+
+void join (std::vector<int>& arr, int l, int m, int r) {
```

Comment:
Extra space before ( in function call  [whitespace/parens] [4]

*Origin: CPPLintBear, Section: `all.cpplint`.*